packages/workbox-webpack-plugin/src/lib/utils/copy-workbox-sw.js

```diff
@@ -23,7 +23,9 @@ module.exports = (readFileFn) => {
   // TODO: this should also support importing workbox-sw from a CDN
   const buildType = (process.env.NODE_ENV &&
     process.env.NODE_ENV.startsWith('dev')) ? 'dev' : 'prod';
-  const defaultWorkboxSourcePath = require.resolve('workbox-sw');
+  // workbox-sw is installed as a dependency of workbox-build.
+  const defaultWorkboxSourcePath = require.resolve(
```

Comment:
This is a stop-gap, as this code is likely to change to a) support CDNs and b) use the methods in `workbox-build` to copy over not on the `workbox-sw` file, but also any other dependencies that are required.